### PR TITLE
Add ReaderT value alias

### PR DESCRIPTION
**Pull request by adelbertc**

### CODE REVIEW

**Comment by xuwei-k** [comment]
:+1:

File: core/src/main/scala/scalaz/package.scala [hunk 1]
```diff
@@ -114,6 +114,7 @@ package object scalaz {
   type |>=|[G[_], F[_]] = MonadPartialOrder[G, F]
 
   type ReaderT[F[_], E, A] = Kleisli[F, E, A]
+  val ReaderT = Kleisli
```
Comment (jedws) [inline core/src/main/scala/scalaz/package.scala]: should add one for Reader as well

Comment (adelbertc) [inline core/src/main/scala/scalaz/package.scala]: Hmm we have this https://github.com/scalaz/scalaz/blob/b55570be26de7051d9ed8904d7effeae34ed52cf/core/src/main/scala/scalaz/package.scala#L124-126

or did you have something else in mind?

Comment (jedws) [inline core/src/main/scala/scalaz/package.scala]: yes, ignore me sorry

**Pull request merged**